src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -210,7 +210,7 @@ export default (opts) => {
     setLoader: gravityLoader((id) => `set/${id}`),
     setsLoader: gravityLoader("sets", {}, { headers: true }),
     showLoader: gravityLoader((id) => `show/${id}`),
-    showsLoader: gravityLoader("shows"),
+    showsLoader: gravityLoader("shows", {}, { headers: true }),
```

Comment:
Any reason to not use the `showsWithHeadersLoader` that already exists on L214?

Reply:
lol, guess I missed it. Probably best to just pull off 248a1f3 in that case and swap to use that